src/requires.jl

```diff
@@ -15,40 +15,38 @@ function __init__()
     end)
 
     @require Colors="5ae59095-9a9b-59fe-a467-6f913c188581" begin
-        fixnotation(c::Float64) = round(c, digits = 4)
+        function PGFPlotsX._rgb_for_printing(c::Colors.Colorant)
+            rgb = convert(Colors.RGB{Float64}, c)
+            round.((red(rgb), green(rgb), blue(rgb)); digits = 4) # somewhat arbitrary choice of accuracy
+        end
+        
         function PGFPlotsX.print_opt(io::IO, c::Colors.Colorant)
-            rgb = convert(Colors.RGB, c)
-            _rgb_64 = convert.(Float64, (rgb.r, rgb.g, rgb.b))
-            rgb_64 = fixnotation.(_rgb_64)
+            rgb_64 = _rgb_for_printing(c)
             print(io, "rgb,1:",
                   "red,"  , rgb_64[1], ";",
                   "green,", rgb_64[2], ";",
                   "blue," , rgb_64[3])
         end
-
+    
```

Comment:
don't make a separate commit for this, but next time please try to remove extra space accidentally introduced

Reply:
So a space got deleted and a space got introduced? So no real change. I'm not sure how I caused this, I'm sure I did, but I'm confused how and ultimately why this matters?

Reply:
No, you added 4 spaces on the empty line. No big deal, but if possible, we like to keep code tidy.

Reply:
An empty line was replaced by a line with spaces.

Reply:
I see I see I see...